components/Navigation.js

```diff
@@ -2,28 +2,28 @@ import Link from "next/link";
 import { useRouter } from "next/router";
 import styled from "styled-components";
 import Image from "next/image";
-import BookmarkIcon from "assets/bookmarkicons/BookmarkPinIcon.svg";
+import BookmarkIcon from "assets/bookmarkicons/Bookmark-filled.svg";
 
 export default function Navigation() {
   const { pathname } = useRouter();
-  const isActive = (page) => pathname === page;
+  const $isActive = (page) => pathname === page;
 
   return (
     <StyledNavigationBar>
       <StyledLink href="/bookmarks">
-        <IconWrapper isActive={isActive("/bookmarks")}>
+        <IconWrapper $isActive={$isActive("/bookmarks")}>
           <StyledBookmarkIcon
-            src="/BookmarkPinIcon.svg"
+            src="/Bookmark-filled.svg"
             alt="Bookmark Icon"
-            width={24}
-            height={24}
+            width={8}
+            height={8}
```

Comment:
You are defining `width` and `height` in your styled component already. Also the `src` attribute is not needed here as you are importing the SVG directly as `BookmarkIcon`.